projects/videolan.org/x264/package.yml

```diff
@@ -0,0 +1,36 @@
+distributable:
+  url: https://code.videolan.org/videolan/x264/-/archive/master/x264-master.tar.gz
+  strip-components: 1
+
+versions:
+  - 0.164
```

Comment:
ideally there's a page we can scrape /cc @jhheider

Reply:
https://artifacts.videolan.org/x264/release-macos/ maybe. won't help us generate a URL though...

Reply:
Probably piggybacking on debian is the way to go: http://deb.debian.org/debian/pool/main/x/x264/

Reply:
Thanks @jhheider 
In the last commit, I added a versioned link. But version salvage fails due to extra text (e.g. +gitbaee400)

Reply:
Good point. Who writes software with no releases?